src/instrument/devices/individual_slits.py

```diff
@@ -0,0 +1,67 @@
+"""
+Individual slits of JJ slit
+"""
+
+import logging
+
+from ophyd import Device
+from ophyd import EpicsMotor
+from ophyd import FormattedComponent as FCpt
+
+logger = logging.getLogger(__name__)
+logger.info(__file__)
+
+
+class IndividualSlits(Device):
```

Comment:
There is an [`MotorBundle`](https://github.com/bluesky/ophyd/blob/84d9e3dfb0e3c1743f211f21954eaa17953b4c1c/ophyd/epics_motor.py#L381) object that is often overlooked.  It might be appropriate to switch to that here from `Device`.  It's docs state:

> This provides better default behavior for 'hints'.